addons/arsenal/stringtable.xml

```diff
@@ -782,6 +786,7 @@
         </Key>
         <Key ID="STR_ACE_Arsenal_enableIdentityTabsSettings">
             <English>Enable the faces / voices / insignias tabs</English>
+            <German>Aktiviere die Gesichert, Stimmen und Abzeichen Übersicht</German>
```

Comment:
```suggestion
            <German>Aktiviere die Gesichter, Stimmen und Abzeichen Übersicht</German>
```

Maybe "Aktiviert" is better? Maybe the english one should be changed to "Enables"

Reply:
Gesichter-, Stimmen- und Abzeichenübersicht